docs/3.x.x/plugin-development/quick-start.md

```diff
@@ -7,28 +7,36 @@ To facilitate the development of a plugin, we drastically reduce the amount of c
 To setup the development environment please **follow the instructions below:**
 
 1. [Fork the repository](https://github.com/strapi/strapi) to your own GitHub account.
-2. Clone it to your computer `git clone git@github.com:strapi/strapi.git`.
-3. Run `npm run setup` at the root of the directory.
+2. Clone it to your computer `git clone git@github.com:strapi/strapi.git`. (or your fork)
+3. Run `cd strapi && npm run setup`
 
 You can run `npm run setup:build` to build the plugins' admin (the setup time will be longer)
 
+Create a development project
+
+1. Create a new project `cd .. && strapi new myDevelopmentProject --dev`.
+2. `cd myDevelopmentProject && strapi start` To start the strapi project
+
 ::: note
 If the installation failed, please remove the global packages related to Strapi. The command `npm ls strapi` will help you to find where your packages are installed globally.
 :::
 
 ## Plugin development Setup
 
-Create a development project
+In a new terminal window:
+
+1. Generate a new plugin: `cd /path/to/myDevelopmentProject && strapi generate:plugin my-plugin`
+2. Install admin dependencies and build: `cd /admin && npm setup`
```

Comment:
I'm just wondering if you added the `npm run setup` command to avoid the link of the `strapi-helper-plugin` in the generated plugin?

Reply:
@soupette Partially, yes. I found the explicit linkage to not work for me, coming from a fresh install. The only thing that got the admin server running was using the npm scripts to handle that linkage for me, and in this order. Mostly, I am updating this based on the work already done by you in your comments in #1116, so if you feel there is inaccuracies, please let me know and I can try and test them out when I am able.

Reply:
@KempfCreative your steps looks good the `setup` scripts in the admin takes care of linking the `strapi-helper-plugin` .